### PR TITLE
Feature/initproject

### PR DESCRIPTION
**Pull request by nottherealalanturing**
## Describe your changes
Setting up project repo with CRA
- Added linters
- Added Routes with react router
- Added Book, Header, Form and category components

### What kind of change does this PR introduce?
New Features.

### In which branch do you want the changes to merge?

dev.

### Documentation updated ?

Yes

### Checklist

- [ ] No ES Lint Error
- [ ] No Style Lint Error
- [ ] Unit test case update

###Screenshot

Screen Shot 2022-06-04 at 14 25 38

### CODE REVIEW

File: src/pages/Books.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const Books = () => {
+  return (
+    <div className="books">
+      <ul>
+        {Array(5)
+          .fill('')
+          .map((book, index) => {
+            return <Book name={index} />;
+          })}
```
Comment (brytebee) [inline src/pages/Books.jsx]: - [x] **Required:** I see you have written the component for for the form but probably forgot🥵  to import and render it on the book page. It is required to have a form, kindly include it.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,60 @@
...
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+Give a ⭐️ if you like this project!
+```
```
Comment (brytebee) [inline README.md:61]: - [x] **Required:** Your documentation is missing some key sections. Kindly include the project description, the acknowledgement and/or license as [templated here](https://github.com/microverseinc/readme-template)

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @nottherealalanturing👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- The project is initialized using `create-react-app`✔️
- Renders a list of books using the Book component.✔️
- Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations.✔️
* Has 2 `<Route>`s, one for the BookList component and the other for the Categories component.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- Work to improve on your commit messages using [this guideline.](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53)

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @nottherealalanturing , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**